userspace/libsinsp/test/test_utils.h

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (C) 2021 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace test_utils {
+
+std::string to_null_delimited(std::vector<std::string> list);
```

Comment:
If you have time I will add a little comment to let people understand that you put a terminator after every single string and not only at the end. Obviously it is clear from the code but not so much from the name... just one line to put lazy developers in the right direction is good for me! This is a minor so fill free to ignore it :)

Reply:
Sure, comment added!